packages/core/src/oidc/grants/token-exchange/index.ts

```diff
@@ -190,11 +197,6 @@ export const buildHandler: (
   ctx.oidc.entity('AccessToken', accessToken);
   const accessTokenString = await accessToken.save();
 
-  // Consume the subject token
-  await updateSubjectTokenById(subjectToken.id, {
```

Comment:
The `subjectToken` consumption has been changed here.  Please confirm if this is expected.
E.g. In your latest implementation, if someone is trying to access the organization info, an error will thrown, but the subject token will be marked as consumed already.